en/docs/get-started/quick-start-guide.md

```diff
@@ -1,124 +1,134 @@
 # Quick Start Guide
 
-Choreo is a full lifecycle cloud-native developer platform that enables your developers to create, deploy,  run, and govern APIs, integrations, and microservices on Kubernetes.
+Choreo is a full lifecycle cloud-native developer platform that enables developers to create, deploy, run, and govern APIs, integrations, and microservices on Kubernetes.
 
-This guide walks you through the following:
+In this quick start guide, you will explore how to expose a service endpoint as a REST API via Choreo and securely consume the API from a web application. 
+You will build a simple reading list web application with a sign-in page and functionality to interact with a secure backend REST API. You will use Asgardeo, WSO2's Identity as a Service (IDaaS) solution, to secure user authentication to the web application. The application will allow users to sign in and view the reading list. On signing in, a user can add books to a reading list and delete books from the reading list. The application will also allow users to sign out of the application.
```

Comment:
we are repeating signing in here..

Reply:
Rephrased the first sentence to remove the sign-in page aspect.